phoenix-core/src/it/java/org/apache/phoenix/end2end/ParameterizedIndexUpgradeToolIT.java

```diff
@@ -87,9 +87,9 @@ public class ParameterizedIndexUpgradeToolIT extends BaseTest {
     private static final String [] TABLE_LIST_NAMESPACE_SIMPLIFIED = new String[1];
 
     private static final String [] TRANSACTIONAL_TABLE_LIST = new String[1];
-
+    private static String tmpPath = System.getProperty("java.io.tmpdir");
     private static String INPUT_LIST = "";
-    private static final String INPUT_FILE = "/tmp/input_file_index_upgrade.csv";
+    private static final String INPUT_FILE = tmpPath + "/input_file_index_upgrade.csv";
```

Comment:
This is where you use the the saved tmpdir.

You should instead use tmpPath which is read AFTER minicluster is started, which can be a different value (depending onm HBase / Hadoop version)